docs/how-to/deploy/using-docker-compose.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Install and Deploy TiKV Using Docker Compose
```

Comment:
I think all these documents should probably end up on tikv.org instead, what do you think?

Reply:
Yeah. I agree.  I think we can finish updating the 3.0 topics for the release. And after that (very soon), we can start properly migrating it.